src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
What do you mean exactly? I got VSCode setup (and I'm loving it), I did what you said:

> you can test by removing it, then holding alt and clicking on fetchCollectorProfile.

and it didn't seem to do anything?